### PR TITLE
Refactor/post functionality

### PR DESCRIPTION
**Pull request by hvohr**
What I did: Implemented the post functionality so that when a user saves a recipe, that recipe goes to the server and it present on refresh for that user. I also refactored another function to correspond with post insertion.

Did this break anything?
- [x] No

Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  The code is DRY. If it's not, I tried my best

What's next? Add iteration 6 and clean up README

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,12 +1,46 @@
-const getData = (type) => {
-  if (!['ingredients', 'users', 'recipes'].includes(type)) {
-    throw new Error("check spelling")
+
+const getRecipes = () => {
+  return fetch('http://localhost:3001/api/v1/recipes')
```
Comment (andreasorensen) [inline src/apiCalls.js:3]: Awesome job on the fetch refactor Hollis! We're getting WOW for sure!

File: src/domUpdates.js [hunk 2]
```diff
@@ -172,11 +185,17 @@ function showSearchResults() {
           </div>`;
       });
     };
-  });
 };
 
-const checkCurrentSavedRecipes= event => {
-  const recipeFound = currentUser.recipesToCook.some(recipe => recipe.name.includes(event.target.parentElement.id));
+const checkCurrentSavedRecipes = event => {
+  let recipeID;
```
Comment (andreasorensen) [inline src/domUpdates.js:191]: Glad you were able to make this work!

**Review by andreasorensen — APPROVED** [review]
Thanks for all your hard work!

File: src/domUpdates.js [hunk 3]
```diff
@@ -25,29 +25,47 @@ const searchButton = document.querySelector('.search-button')
 
 //Event Listeners
 
+const startFetch = () => {
```
Comment (fernandorobles97) [inline src/domUpdates.js:28]: Great job implementing a data model from the fetched data!

**Review by fernandorobles97 — APPROVED** [review]

**Review by Denzel-Turambi — APPROVED** [review]
Nicely done! Great work on refactor the get fetch requests, and doing the post fetch request

**Pull request merged**